ci-operator/step-registry/ipi/conf/nutanix/extended/ipi-conf-nutanix-extended-ref.yaml

```diff
@@ -0,0 +1,29 @@
+ref:
+  as: ipi-conf-nutanix-extended
```

Comment:
From the step's contents, per my understanding, it is covering 3 of test points:
```
bootType
categories
project
```
I suggest to create 3 separate steps to configure them, but not combine all of them together in one step.